alicloud/resource_alicloud_polardb_cluster.go

```diff
@@ -0,0 +1,454 @@
+			return polarDBClient.ModifyDBClusterMaintainTime(request)
+		})
+		if err != nil {
+			return WrapErrorf(err, DefaultErrorMsg, d.Id(), request.GetActionName(), AlibabaCloudSdkGoERROR)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		d.SetPartial("maintain_time")
+	}
+
+	if d.IsNewResource() {
+		d.Partial(false)
+		return resourceAlicloudPolarDBClusterRead(d, meta)
+	}
+
+	if d.HasChange("description") {
+		request := polardb.CreateModifyDBClusterDescriptionRequest()
+		request.RegionId = client.RegionId
+		request.DBClusterId = d.Id()
+		request.DBClusterDescription = d.Get("description").(string)
+
+		raw, err := client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+			return polarDBClient.ModifyDBClusterDescription(request)
+		})
+		if err != nil {
+			return WrapErrorf(err, DefaultErrorMsg, d.Id(), request.GetActionName(), AlibabaCloudSdkGoERROR)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		d.SetPartial("description")
+	}
+
+	if d.HasChange("security_ips") {
+		ipList := expandStringList(d.Get("security_ips").(*schema.Set).List())
+
+		ipstr := strings.Join(ipList[:], COMMA_SEPARATED)
+		// default disable connect from outside
+		if ipstr == "" {
+			ipstr = LOCAL_HOST_IP
+		}
+
+		if err := polarDBService.ModifyDBSecurityIps(d.Id(), ipstr); err != nil {
+			return WrapError(err)
+		}
+		d.SetPartial("security_ips")
+	}
+
+	d.Partial(false)
+	return resourceAlicloudPolarDBClusterRead(d, meta)
+}
+
+func resourceAlicloudPolarDBClusterRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*connectivity.AliyunClient)
+	polarDBService := PolarDBService{client}
+
+	clusterAttribute, err := polarDBService.DescribePolarDBClusterAttribute(d.Id())
+	if err != nil {
+		if NotFoundError(err) || IsExceptedErrors(err, []string{InvalidDBClusterIdNotFound, InvalidDBClusterNameNotFound}) {
+			d.SetId("")
+			return nil
+		}
+		return WrapError(err)
+	}
+
+	cluster, err := polarDBService.DescribePolarDBCluster(d.Id())
+	if err != nil {
+		if NotFoundError(err) || IsExceptedErrors(err, []string{InvalidDBClusterIdNotFound, InvalidDBClusterNameNotFound}) {
+			d.SetId("")
+			return nil
+		}
+		return WrapError(err)
+	}
+
+	ips, err := polarDBService.GetSecurityIps(d.Id())
+	if err != nil {
+		return WrapError(err)
+	}
+
+	d.Set("security_ips", ips)
+
+	d.Set("vswitch_id", clusterAttribute.VSwitchId)
+	d.Set("pay_type", clusterAttribute.PayType)
+	d.Set("id", clusterAttribute.DBClusterId)
+	d.Set("description", clusterAttribute.DBClusterDescription)
+	d.Set("db_type", clusterAttribute.DBType)
+	d.Set("db_version", clusterAttribute.DBVersion)
+	d.Set("maintain_time", clusterAttribute.MaintainTime)
+	d.Set("zone_ids", clusterAttribute.ZoneIds)
+	d.Set("cluster_charge_type", getChargeType(clusterAttribute.PayType))
+	d.Set("cluster_network_type", clusterAttribute.DBClusterNetworkType)
```

Comment:
I think the schema does have a parameter named as  `cluster_network_type `. Please keep the output sync with schema.